R/epidatacall.R

```diff
@@ -198,6 +198,14 @@ fetch_args_list <- function(
   )
 }
 
+#' @export
+print.fetch_args <- function(x, ...) {
+  stopifnot(inherits(x, "fetch_args"))
+  cli::cli_h1("<fetch_args> object:")
+  # Print all non-class fields.
+  print(x[attr(x, "names")])
```

Comment:
issue?: I think this probably does both printing via messages and via regular prints to stdout.  This causes headaches when you want to build .Rmds and/or `capture.output`.  See cmu-delphi/epipredict#277.

Reply:
Used `cli::cli_dl` to print name-value pairs. Most `cli` functions drop item names.